src/components/Footer/footer.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { render } from '@testing-library/react';
+import i18n from '../../i18n';
+import Footer from './Footer';
+
+describe('Footer test', () => {
+  it('Footer main text', async () => {
+    const { getByText } = render(
+      <I18nextProvider i18n={i18n}>
```

Comment:
You don't have to use i18n unless you define a variable that contains text in two different languages. Check out you resources.json file in the i18n project folder.